### PR TITLE
Update GlobalStyle to include fontfaces from @twreporter/react-components

### PR DESCRIPTION
**Pull request by taylrj**
Address [TWREPORTER-318](https://twreporter-org.atlassian.net/browse/TWREPORTER-318?atlOrigin=eyJpIjoiNjBiZDgyYTlkNWQ5NDA4ODgxNzQ5NTRmM2EwMjE2MzkiLCJwIjoiaiJ9)

https://github.com/twreporter/twreporter-npm-packages/pull/297 is required by this PR.

### CODE REVIEW

**Comment by taylrj** [comment]
[166f264](https://github.com/twreporter/twreporter-react/pull/2216/commits/166f264bd72cb3cc096fc0a6622a544249401383) adds defaultFallback to provide fallback font family when font loaded failed.

**Comment by taylrj** [comment]
Updated with `preload` font files link.

File: src/html.js [hunk 1]
```diff
@@ -67,6 +71,16 @@ export default class Html extends PureComponent {
 
           <link href="/asset/favicon.png" rel="shortcut icon" />
           <link rel="stylesheet" href="/asset/normalize.css" />
+          <link rel="stylesheet" href="/asset/normalize.css" />
```
Comment (duidae) [inline src/html.js:74]: mostly looking good, only this duplicate `<link rel="stylesheet" href="/asset/normalize.css" />`, is it copied by accident?

Comment (taylrj) [reply src/html.js]: Oops, it was an accident...
Updated in https://github.com/twreporter/twreporter-react/pull/2216/commits/d32a02e6f38620564a481711b5425423d60fc72f.

**Review by duidae — APPROVED** [review]
LGTM

**Comment by taylrj** [comment]
- Rebase with upstream/master
- Add the required @twreporter npm packages

**Pull request merged**